data/nfv/id_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Why would you like to have hard-coded SSH keys? Shouldn't we avoid this kind of things?
I would suggest creating and authorizing keys on-demand and not add these here.

Reply:
I could use ssh-keygen in the test to generate the key inside the test, but then I can't get put it into the authorized_keys of the other server. Doing it this way makes things much easier.

Reply:
Now I am not using hard-coded keys :)